publish/directory/openapi.json

```diff
@@ -2118,20 +2118,7 @@
   "servers": [
     {
       "description": "Aserto Directory service",
-      "url": "https://directory.prod.aserto.com"
-    },
-    {
-      "description": "Local directory service using Topaz",
-      "url": "https://localhost:{port}",
-      "variables": {
-        "port": {
-          "default": "9393",
-          "enum": [
-            "8383",
-            "9393"
-          ]
-        }
-      }
+      "url": "{{.Scheme}}://{{.Host}}"
```

Comment:
One thing I didn't think about when we spoke yesterday is that in addition to serving the spec from topaz and directory instances, we also upload it to [aserto.readme.io](https://aserto.readme.io/reference/directoryv3assertionset) from [within CI](https://github.com/aserto-dev/openapi-directory/blob/main/.github/workflows/ci.yaml#L105).
And I think that we do want that version to include both `prod` and `localhost` servers because the readme.io UI lets you select which server to target.

We should think about the right way to achieve that.